config/contributors.txt

```diff
@@ -0,0 +1,6 @@
+johnwillard
+sgthunk
+spacedragon00
+cyotheprotogen
+guillaumeprata
+helianthus
```

Comment:
```suggestion
helianthus
horatio22
windarkata
IVlalygos
theswain
```

You should add in the rest of the contributors with mentor perms

Reply:
IVlalygos

cant commit your suggestion because its supposed to be all lowercase

Reply:
that was copied straight from the mentor.txt file so I don't really think it makes a difference. There are a few names in there with capitals.

Reply:
deal, with it